tests/framework/MCMC/test_1D.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="silent">
+  <TestInfo>
+    <name>framework/MCMC.MetropolisOneDim</name>
+    <author>wangc</author>
+    <created>2020-12-20</created>
+    <classesTested>MCMC.Metropolis</classesTested>
+    <description>
+      Test the Markov Chain Monte Carlo algorithm: Metropolis Sampling
```

Comment:
can you expand the description a bit more?

Reply:
updated